web/src/features/public-api/types/models.ts

```diff
@@ -29,11 +30,12 @@ const APIModelDefinition = z
     inputPrice: z.number().nonnegative().nullable(),
     outputPrice: z.number().nonnegative().nullable(),
     totalPrice: z.number().nonnegative().nullable(),
-    unit: APIModelUsageUnit,
+    unit: APIModelUsageUnit.nullish(),
```

Comment:
This is probably also breaking if customers rely on it. Why does it become nullish here?